huxley/api/urls.py

```diff
@@ -27,7 +27,8 @@
     url(r'^schools/?$', views.school.SchoolList.as_view(), name='school_list'),
     url(r'^schools/(?P<pk>[0-9]+)/?$', views.school.SchoolDetail.as_view(), name='school_detail'),
     url(r'^schools/(?P<pk>[0-9]+)/assignments/?$', views.school.SchoolAssignments.as_view(), name='school_assignments'),
-    url(r'^schools/(?P<pk>[0-9]+)/invoice/?$', views.school.SchoolInvoice.as_view(), name='school_invoice')
+    url(r'^schools/(?P<pk>[0-9]+)/invoice/?$', views.school.SchoolInvoice.as_view(), name='school_invoice'),
+    url(r'^schools/(?P<pk>[0-9]+)/finalize/?$', views.school.SchoolFinalize.as_view(), name='school_finalize')
```

Comment:
Add a trailing comma at the end of this URI declaration, so we don't have to modify the line when more are added.